lib/src/rules/unsafe_html.dart

```diff
@@ -33,20 +33,36 @@ var script = ScriptElement()..src = 'foo.js';
 ```
 ''';
 
-extension on DartType? {
-  /// Returns whether this type extends [className] from the dart:html library.
-  bool extendsDartHtmlClass(String className) =>
-      extendsClass(className, 'dart.dom.html');
-}
-
 class UnsafeHtml extends LintRule {
+  // static const LintCode attributeCode = LintCode(
```

Comment:
Should this be... removed?

Reply:
See the TODO on line 77. We want to clean up the lint codes, and I'm proposing that these declarations be used when we do so. (In all of the other lints the codes are in the linter rather than the visitor, and I think we should follow the same convention here.)